pillar/fluentd/server.sls

```diff
@@ -73,14 +73,6 @@ fluentd:
             - format: json
             - port: 9999
             - keep_time_key: 'true'
-        - directive: source
-          attrs:
-            - '@type': secure_forward
```

Comment:
If we remove this entire block then how are we going to continue to receive these logs?